Kitodo/src/main/resources/messages/messages_en.properties

```diff
@@ -186,7 +187,9 @@ benutzergruppen=User groups
 benutzergruppenHinzufuegen=Add user groups
 benutzerkonfiguration=User configuration
 benutzerprofilLaden=Load user profile
-benutzerverwaltung=User management
+benutzerverwaltung=Users management
+benutztShell=Uses shell
+berechnungen=Calculations
```

Comment:
This "new" translation is not used and was maybe removed by Beatrycze in a precious pull request. So do not add it again.